tests/test_filters/test_callback_data.py

```diff
@@ -148,6 +148,13 @@ class MyCallback3(CallbackData, prefix="test3"):
         assert MyCallback3.unpack("test3:experiment:42") == MyCallback3(bar=42)
         assert MyCallback3.unpack("test3:spam:42") == MyCallback3(foo="spam", bar=42)
 
+        class MyCallback4(CallbackData, prefix="test4"):
+            foo: Optional[str] = ''
```

Comment:
Please, use commands from contribution guide:
https://docs.aiogram.dev/en/dev-3.x/contributing.html#format-the-code-code-style

Reply:
```suggestion
            foo: Optional[str] = ""
```